packages/editor/src/bindings/pattern-overrides.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+export default {
+	name: 'core/pattern-overrides',
+	label: __( 'Pattern Overrides' ),
```

Comment:
Shouldn't we use the `_x()` like we do on the PHP side ([example](https://github.com/WordPress/wordpress-develop/blob/f8b9b2abd742e4c8b4a938d4fdd226a2c7d1e794/src/wp-includes/block-bindings/pattern-overrides.php#L40-L41)). If yes, that would also apply to the `post-meta` source.

Reply:
They should be exactly the same as on the server. Good catch 👍

Reply:
Good catch 🙂 Changed in [this commit](https://github.com/WordPress/gutenberg/pull/58787/commits/228c14277b04360a45c9bb5a203c85a834c0d958).